api/CanvasRenderingContext2D.json

```diff
@@ -0,0 +1,3859 @@
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "path": {
+          "__compat": {
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": true,
+                "notes": "This feature is behind a feature flag. Set the flag <code>ExperimentalCanvasFeatures</code> to <code>true</code> to enable it."
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "39",
+                "notes": "This feature is behind a feature preference setting. In about:config, set <code>canvas.hitregions.enabled</code> to <code>true</code>."
+              },
+              "firefox_android": {
+                "version_added": "30"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "fillRule": {
+          "__compat": {
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": true,
+                "notes": "This feature is behind a feature flag. Set the flag <code>ExperimentalCanvasFeatures</code> to <code>true</code> to enable it."
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "other hit region options": {
```

Comment:
No identifier with spaces here. Please use underscores, and add `"description": "Other hit region options"`.